statics/workflows/ping-mesh.yaml

```diff
@@ -0,0 +1,103 @@
+---
+UUID: "1d81b961-5d62-49b7-42f3-c6afc616ff98"
+name: "PingMesh(ICMP/TCP/UDP)"
+description: "Check Connectivity from Multiple Source to Single Destination"
+parameters:
+  - name: source
+    description: Source node
+    type: node
+  - name: destination
+    description: Destination node
+    type: node
+  - name: protocol
+    description: Protocol
+    type:  choice
+    default: ICMPv4
+    values:
+      - description: "Protocol : ICMPv4/Echo request"
+        value: ICMPv4
+      - description: "Protocol : TCP/IPv4"
+        value: TCP
+      - description: "Protocol : UDP/IPv4"
+        value: UDP	
+source: |
+    function PingMesh(from, to, protocol) {
+      var dict = {},
+            From = {},
+            sources = [],
+            captureuuids = [],
+            i = 1,
+            randvalue = Math.floor(Math.random() * 4096);
```

Comment:
why only limit the random on 4k ? why not 16bits/64k (icmpID ; tcp/udp Port size)